Containers/nextcloud/entrypoint.sh

```diff
@@ -258,26 +258,21 @@ DATADIR_PERMISSION_CONF
             if [ "$INSTALL_LATEST_MAJOR" = yes ]; then
                 php /var/www/html/occ config:system:set updater.release.channel --value=beta
```

Comment:
still want to stay on beta?

Reply:
Ah right, forgot to remove that

Reply:
done with https://github.com/nextcloud/all-in-one/pull/3915/commits/6412aa3e0f4262e5fa6b163be7efb3bbd2ed5e2e